src/blockchain/start-augur-listeners.ts

```diff
@@ -27,11 +27,11 @@ export function startAugurListeners(db: Knex, augur: Augur, callback: (blockNumb
       Transfer: makeLogListener(db, augur, "LegacyReputationToken", "Transfer"),
       Approval: makeLogListener(db, augur, "LegacyReputationToken", "Approval"),
     },
-  }, (blockNumber: Int256): void => {
+  }, (block: Block): void => {
```

Comment:
FYI: If we ever get full type defenitions for `augur.js` you'll be able to change this to `block => { ... }` and type inference will figure out the rest.

Reply:
Actually, we have defined the type for this, so i can technically remove it. Is there a best practice for when you do so? just, whenever possible?

Reply:
https://github.com/AugurProject/augur.js/blob/91539115d4c32648df3cb77b65f1859fd456629f/index.d.ts#L169

Reply:
I'm personally a big fan of letting type inference do its job.  However, I also am a big fan in running TS in ultra-strict mode which doesn't allow for implicit `any` types so you get a complaint when the type inferer _can't_ do its job.  Without strict mode, I would be hesitant to follow my usual pattern of "remove type annotations where possible" since it is too easy to fall back into "infer `any`" mode.